cmd/disk/main.go

```diff
@@ -35,13 +36,13 @@ const (
 )
 
 var (
-	configPath       = flag.String("config", defaultConfigPath, "server config file path")
-	driverName       = flag.String("drivername", defaultProvisionName, "name of the driver")
-	endpoint         = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
-	maxVolume        = flag.Int64("maxvolume", 10, "Maximum number of volumes that controller can publish to the node.")
-	nodeId           = flag.String("nodeid", "", "If driver cannot get instance ID from /etc/qingcloud/instance-id, we would use this flag.")
-	retryIntervalMax = flag.Duration("retry-interval-max", 2*time.Minute, "Maximum retry interval of failed deletion.")
-	retryTimesMax    = flag.Int("retry-times-max", 10, "Maximum retry times of failed detach volume.")
+	configPath          = flag.String("config", defaultConfigPath, "server config file path")
+	driverName          = flag.String("drivername", defaultProvisionName, "name of the driver")
+	endpoint            = flag.String("endpoint", "unix://tmp/csi.sock", "CSI endpoint")
+	maxVolume           = flag.Int64("maxvolume", 10, "Maximum number of volumes that controller can publish to the node.")
+	nodeId              = flag.String("nodeid", "", "If driver cannot get instance ID from /etc/qingcloud/instance-id, we would use this flag.")
+	retryIntervalMax    = flag.Duration("retry-interval-max", 2*time.Minute, "Maximum retry interval of failed deletion.")
```

Comment:
should we remove retryIntervalMax? ie. remove the retries and hand over the control to csi sidecar?

Reply:
Yes, I think it should be removed, I'll do that in another PR.